resources/texts/change_log.txt

```diff
@@ -64,6 +89,14 @@ We now support Apple M1 chips
 - Updated Snapmaker 2 End-Gcode, contributed by Rolzad73
 - Updated the Tinyboy Fabricator printer definitions, contributed by reibuehl
 - Updated the Creality Ender 5 printer profile, contributed by Rakhmanov
+														  
+* Community translations:
+- Updated the Brazilian Portuguese translation, contributed by Patola
+- Updated the Czech translation, contributed by sustmi
+- Updated some of the Chinese translation, contributed by iBral
+
+<i>From version 5.0 onwards - Ultimaker Cura is only supported on operating systems actively maintained by their software manufacturer or community. This means Windows 7 and MacOS 10.14 will no longer be supported. Technically this means Ultimaker will stop testing and developing for such operating systems. However, even though it is no longer supported, there is still a high likelihood the application keeps functioning.
```

Comment:
We know for very sure that Windows 7 does not work, and AFAIK neither does MacOS 10.14. So that last sentence is somewhat meaningless.